services/order.py

```diff
@@ -0,0 +1,38 @@
+from db.models import Order, Ticket, User
+from django.db import transaction
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> None:
+    with transaction.atomic():
+        try:
+            user = User.objects.get(username=username)
+        except User.DoesNotExist:
+            raise ValueError("User does not exist")
+
+        order = Order.objects.create(user=user)
+
+        if date:
+            order.created_at = date
+
+        order.save()
+
+        for ticket in tickets:
+            Ticket.objects.create(
+                row=ticket["row"],
+                seat=ticket["seat"],
+                movie_session_id=ticket["movie_session"],
+                order=order
+            )
```

Comment:
The `create_order` function assumes that each ticket dictionary contains the keys 'row', 'seat', and 'movie_session'. Consider adding validation to ensure these keys are present to prevent potential `KeyError` exceptions.